### PR TITLE
issue-100 Update instructions

### PR DESCRIPTION
**Pull request by docktermj**
# Pull request questions

## Which issue does this address

ISSUE-100

### CODE REVIEW

**Comment by docktermj** [comment]
#100

File: README.md [hunk 1]
```diff
@@ -160,97 +160,95 @@ Other command-line options may be useful to you as well.  Execute
 `java -jar target/senzing-api-server-1.7.2.jar -help` to obtain a help message
 describing all available options.  For example:
 
-  ```console
+```console
+$ java -jar target/senzing-api-server-1.7.2.jar -help
+java -jar senzing-api-server-1.7.2.jar <options>
```
Comment (barrycaceres) [inline README.md]: Version number of 1.7.2 should probably change

File: README.md [hunk 2]
```diff
@@ -48,83 +48,86 @@ You will also need the Senzing "g2.jar" file installed in your Maven repository.
...
- 1. Determine your `${SENZING_VERSION}` version number:
-    - Locate your `g2BuildVersion.json` file:
-        - Linux: `${SENZING_G2_DIR}/data/g2BuildVersion.json`
-        - Windows: `${SENZING_G2_DIR}\data\g2BuildVersion.json`
-    - Find the value for the `"VERSION"` property in the JSON contents.
-      Example:
+1. Determine your `SENZING_G2_JAR_VERSION` version number:
+    1. Locate your `g2BuildVersion.json` file:
+        1. Linux: `${SENZING_G2_DIR}/g2BuildVersion.json`
+        1. Windows: `${SENZING_G2_DIR}\g2BuildVersion.json`
```
Comment (barrycaceres) [inline README.md]: Is this true on Windows?  Did the MSI installer really change where it puts that file?  I thought the change was only for the RPM's

File: README.md [hunk 3]
```diff
@@ -160,8 +160,13 @@ Senzing REST API Server using an entity repository from the
 for version compatibility and usage information.*
 
 Other command-line options may be useful to you as well.  Execute
-`java -jar target/senzing-api-server-1.7.2.jar -help` to obtain a help message
-describing all available options.  For example:
+
+```console
+java -jar target/senzing-api-server-1.7.2.jar -help
```
Comment (barrycaceres) [inline README.md]: Do we want to update the version number from 1.7.2 ??

File: README.md [hunk 4]
```diff
@@ -44,9 +44,9 @@ documents the available API methods, their parameters and the response formats.
 To build the Senzing REST API Server you will need Apache Maven (recommend version 3.5.4 or later)
 as well as Java 1.8.x (recommend version 1.8.0_171 or later).
 
-You will also need the Senzing "g2.jar" file installed in your Maven repository.
-The Senzing REST API Server requires version 1.7.x or later of the Senzing API
-and Senzing App.  In order to install g2.jar you must:
+You will also need the Senzing `g2.jar` file installed in your Maven repository.
+The Senzing REST API Server requires version 1.7.x or later of the Senzing API and Senzing App.
```
Comment (barrycaceres) [inline README.md]: should be "version 1.10.x or later" instead of "1.7.x or later"

**Comment by docktermj** [comment]
#100

**Pull request merged**